doc/how-to/recover.md

```diff
@@ -0,0 +1,63 @@
+# MicroCloud Cluster Recovery
+
+Each MicroCloud service uses the [Dqlite](https://dqlite.io/) distributed
```

Comment:
You'll need to add Dqlite to the wordlist here as well - there's no automatic update (unfortunately :disappointed: ).

Reply:
K, wasn't sure if there was a process for pulling in starter pack changes.